client/flutter/lib/pages/main_pages/routes.dart

```diff
@@ -16,8 +16,7 @@ import 'package:who_app/pages/travel_advice.dart';
 
 class Routes {
   static final Map<String, WidgetBuilder> map = {
-    '/': (context) =>
-        AppTabRouter(AppTabRouter.defaultTabs, AppTabRouter.defaultNavItems),
+    '/': (context) => AppTabRouter(),
```

Comment:
why did you revert this change?

Reply:
Misunderstood comments above -- just pushed fix